plugins-builtin/src/test/scala/za/co/absa/enceladus/plugins/builtin/errorsender/mq/KafkaErrorSenderPluginSuite.scala

```diff
@@ -123,35 +124,13 @@ class KafkaErrorSenderPluginSuite extends AnyFlatSpec with TZNormalizedSparkTest
       schemaRegistryUrl = testSchemaRegUrl, clientId = testClientId,
       security = Some(KafkaSecurityParams(testSecurityProtocol, Some(testSaslMechanism))), topicName = testTopicName,
       schemaRegistrySecurityParams = Some(SchemaRegistrySecurityParams(testSchemaRegAuthSource, Some(testSchemaRegAuthUserInfo))))
-
-    errorPlugin.keySchemaRegistryConfig shouldBe Map(
-      SchemaManager.PARAM_SCHEMA_REGISTRY_URL -> testSchemaRegUrl,
-      SchemaManager.PARAM_SCHEMA_REGISTRY_TOPIC -> testTopicName,
-      SchemaManager.PARAM_KEY_SCHEMA_NAMING_STRATEGY -> SchemaManager.SchemaStorageNamingStrategies.TOPIC_NAME,
-      SchemaManager.PARAM_SCHEMA_NAME_FOR_RECORD_STRATEGY -> "dataErrorKey",
-      SchemaManager.PARAM_SCHEMA_NAMESPACE_FOR_RECORD_STRATEGY -> "za.co.absa.dataquality.errors.avro.key.schema",
-      "basic.auth.credentials.source" -> "USER_INFO",
-      "basic.auth.user.info" -> "svc-account:SVC-P4SSW0RD"
-    )
-
-    errorPlugin.valueSchemaRegistryConfig shouldBe Map(
-      SchemaManager.PARAM_SCHEMA_REGISTRY_URL -> testSchemaRegUrl,
-      SchemaManager.PARAM_SCHEMA_REGISTRY_TOPIC -> testTopicName,
-      SchemaManager.PARAM_VALUE_SCHEMA_NAMING_STRATEGY -> SchemaManager.SchemaStorageNamingStrategies.TOPIC_NAME,
-      SchemaManager.PARAM_SCHEMA_NAME_FOR_RECORD_STRATEGY -> "dataError",
-      SchemaManager.PARAM_SCHEMA_NAMESPACE_FOR_RECORD_STRATEGY -> "za.co.absa.dataquality.errors.avro.schema",
-      "basic.auth.credentials.source" -> "USER_INFO",
-      "basic.auth.user.info" -> "svc-account:SVC-P4SSW0RD"
-    )
   }
```

Comment:
Obviously, this is no longer usable like this, because the internals have changed. However, I think it would be unfortunate to forgo the registry settings check entirely, so I suggest:

1 Change above:
```scala
val testSchemaRegUrl = "http://example.com:8081"
```
to
```scala
val testSchemaRegUrl = "mock://example.com:8081"
```
The reason si that for URL starting with `mock://`, Abris will not fail on `java.net.ConnectException: Connection timed out: connect` and the method 
`KafkaErrorSenderPlugin.registerSchemas` can now be tested.

2 Add the following test case:
```scala
  it should "correctly register schemas" in {
    val connectionParams = KafkaErrorSenderPlugin.kafkaConnectionParamsFromConfig(testConfig)

    KafkaErrorSenderPlugin.registerSchemas(connectionParams) match {
      case (_, _, schemaConfig) => // we do not care about `keySchemaId`, `valueSchemaId` assigned
        schemaConfig shouldBe Map(
          "schema.registry.url" -> "mock://example.com:8081",
          "basic.auth.credentials.source" -> "USER_INFO",
          "basic.auth.user.info" -> "svc-account:SVC-P4SSW0RD"
        )
    }
  }
```

Reply:
Thank you, brilliant. Just copy&pasted it, works!